### PR TITLE
Car owner can update car information

### PR DESCRIPTION
**Pull request by johannaanden**
### PT link
[PT link](https://www.pivotaltracker.com/story/show/162394191)

### Actions
- Added update action
- Added destroy action
- Feature test for happy and sad path
- Added success and error message

### CODE REVIEW

File: app/controllers/automobiles_controller.rb [hunk 1]
```diff
@@ -18,6 +18,26 @@ def create
     end
   end
 
+  def edit
+    @automobile = Automobile.find(params[:id])
+  end
+
+  def update
+    @automobile = Automobile.find(params[:id])
+    -if @automobile.update(automobile_params)
```
Comment (oliverochman) [inline app/controllers/automobiles_controller.rb]: remove the dash before the if statement

File: features/car_owner_can_update_car_information.feature [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Feature: Car owner can update car information
+
+    As a car owner
+    In order to modify car information on the site
+    I want to be able to update car information and/or remove car registration from the site
+
+    Background:
+        Given the following user exists
+            | email          |
```
Comment (oliverochman) [inline features/car_owner_can_update_car_information.feature]: fix indentation

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: app/controllers/automobiles_controller.rb (hunk 1 above)
Comment (johannaanden) [reply app/controllers/automobiles_controller.rb]: done

File: features/car_owner_can_update_car_information.feature (hunk 2 above)
Comment (johannaanden) [reply features/car_owner_can_update_car_information.feature]: done

**Review by oliverochman — APPROVED** [review]

**Pull request merged**